andela/healthchecks-invincibles
# #150798556 Teams create dashboard for mission critical jobs

#### What does this PR do?
Users in teams can be able to view jobs according to their departments
#### Description of Task to be completed?
Users should be able to create alerts for a series of third party online business automation applications .eg. shopify, dasheroo .etc.
#### How should this be manually tested?
- Clone the repository `https://github.com/andela/healthchecks-invincibles`
- Change Directory to the project
- Activate the virtual environment
- Run migrations `./manage.py migrate`
- Navigate to `http://127.0.0.1:8000/` to login then proceed to admin `http://127.0.0.1:8000/admin/`
- Make sure to change your profile and make sure to check `Team access allowed`
<img width="509" alt="screen shot 2017-09-21 at 14 08 32" src="https://user-images.githubusercontent.com/15690240/30693008-6d73bde4-9ed6-11e7-8e73-d7cd52d66254.png">
- Add departments still in the admin page
- Visit the page to access the team tab where you can add different teams and members
<img width="1193" alt="screen shot 2017-09-21 at 14 10 44" src="https://user-images.githubusercontent.com/15690240/30693086-be252d0e-9ed6-11e7-9ef3-435e486184fc.png">
- On the checks dashboard, you should be able to navigate to the Department's tab and filter jobs according to the departments
#### What are the relevant pivotal tracker stories?
150798561
#### Screenshots (if appropriate)
<img width="1192" alt="screen shot 2017-09-21 at 14 02 18" src="https://user-images.githubusercontent.com/15690240/30693131-eca4c914-9ed6-11e7-8277-9b1868e845fc.png">
<img width="1223" alt="screen shot 2017-09-21 at 14 02 33" src="https://user-images.githubusercontent.com/15690240/30693132-eca86998-9ed6-11e7-8c3f-99c1135f7924.png">
<img width="625" alt="screen shot 2017-09-21 at 14 02 51" src="https://user-images.githubusercontent.com/15690240/30693130-eca423d8-9ed6-11e7-9d54-d42d6dd25a09.png">
